app/livechat/lib/LivechatInquiry.js

```diff
@@ -100,6 +142,62 @@ if (Meteor.isServer) {
 
 			return this.update(query, update);
 		}
+
+		async getCurrentSortedQueueAsync({ _id, department }) {
```

Comment:
please create a new raw model like here: https://github.com/RocketChat/Rocket.Chat/blob/develop/app/models/server/raw/index.js#L1

Reply:
The current `LivechatInquiry` file is not implemented as a regular model, maybe we can do it later and then move the method you mentioned to the new regular model.